src/components/libraryoptionseditor/libraryoptionseditor.js

```diff
@@ -411,6 +411,12 @@ import template from './libraryoptionseditor.template.html';
             parent.querySelector('.chkEnableEmbeddedEpisodeInfosContainer').classList.add('hide');
         }
 
+        if (contentType !== 'tvshows' || contentType !== 'movies' || contentType !== 'musicvideos' || contentType !== 'mixed') {
```

Comment:
```suggestion
        if (contentType === 'tvshows' || contentType === 'movies' || contentType === 'musicvideos' || contentType === 'mixed') {
```

I think these were reversed?
(checkbox shows for `book`)

Reply:
Oops yeah, I think it's reversed.